prov/efa/src/rdm/rxr_pkt_type_req.c

```diff
@@ -1858,7 +1858,6 @@ void rxr_pkt_proc_eager_rtw(struct rxr_ep *ep,
 		return;
 	}
 
-	rx_entry->cq_entry.flags |= (FI_RMA | FI_WRITE);
```

Comment:
Does `rx_entry->cq_entry.flags` have FI_REMOTE_WRITE` when `rx_entry` was allocated?

Reply:
I see you already mentioned it in your commit message.